AGENTS.md

```
@@ -19,21 +23,21 @@ Use **beads** (`bd` CLI) for persistent task tracking across sessions. This repl
-| `epic`    | Large feature with subtasks          |
-| `chore`   | Maintenance                          |
-
-| Priority | Meaning                                       |
-| -------- | --------------------------------------------- |
-| `0`      | Critical (security, data loss, broken builds) |
-| `1`      | High (major features, important bugs)         |
-| `2`      | Medium (default)                              |
-| `3`      | Low (polish, optimization)                    |
-| `4`      | Backlog                                       |
+| Type | Use For |
+|------|---------|
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe is missing space to the right for style "compact"

(MD060, table-column-style)